include/lldb/Symbol/ClangASTContext.h

```diff
@@ -760,7 +760,8 @@ class ClangASTContext : public TypeSystem {
   GetBitSize(lldb::opaque_compiler_type_t type,
              ExecutionContextScope *exe_scope) override;
 
-  uint64_t GetByteStride(lldb::opaque_compiler_type_t type) override;
+  llvm::Optional<uint64_t>
+  GetByteStride(lldb::opaque_compiler_type_t type) override;
```

Comment:
Did you make this change in llvm.org, too? If not, please do so.

Reply:
This API is only on github.